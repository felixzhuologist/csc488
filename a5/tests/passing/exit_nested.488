{
  write "open 0", newline
  repeat {
    write "open 1", newline
    repeat {
      write "open 2", newline
        repeat {
          write "open 3", newline
          while true do {
            exit
            write "inside 3", newline
          }
          write "close 3", newline
        } until false
        write "close 2", newline
    } until false
    write "close 1", newline
  } until false
  write "close 0", newline
}