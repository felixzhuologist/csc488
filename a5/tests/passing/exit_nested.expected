open 0
open 1
open 2
open 3
close 3
close 2
close 1
close 0
